### PR TITLE
task/RTMD-19-Configure-Dockerfile

### PR DESCRIPTION
**Pull request by bendiggle**
Changed the Dockerfile and run.sh to match GRO.
Added a docker-compose file

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-
-cp -r /app/public/* /public/
-SECURE_PROTOCOL=true npm start
-
+if [ "$NODE_ENV" = "ci" ]
+then echo "starting service"
+  SITEROOT=/rtm
+fi
 
+exec npm start
```
Comment (JoeChapman) [inline run.sh]: See https://github.com/UKHomeOffice/rotm/pull/31/files#diff-3254677a7917c6c01f55212f86c57fbfL35 this script is never run

Comment (bendiggle) [inline run.sh]: @JoeChapman yeah, good point. This is the same for gro. Do you think the script is needed?

Comment (JoeChapman) [inline run.sh]: no harm in keeping to set the SITEROOT I guess. I don't know what SITEROOT is meant to do?

Comment (bendiggle) [inline run.sh]: @JoeChapman I've updated the Dockerfile to run the script. But I can't answer your question on what SITEROOT is meant to do

**Comment by JoeChapman** [comment]
@bendiggle This docker implementation and the docker-compose configuration should be the same as the one in GRO.
@easternbloc and I have been updating the implementation of docker in GRO, so I suggest you catch up with him

**Pull request closed without merging**